Libplanet.Tests/Net/SwarmTest.cs

```diff
@@ -2813,6 +2769,73 @@ public async Task FindSpecificPeerAsync()
             }
         }
 
+        [Fact(Timeout = Timeout)]
+        public async Task DoNotFillMultipleTimes()
+        {
+            Swarm<DumbAction> receiver = _swarms[0];
+            Swarm<DumbAction> sender1 = _swarms[1];
+            Swarm<DumbAction> sender2 = _swarms[2];
+
+            await StartAsync(receiver);
+            await StartAsync(sender1);
+            await StartAsync(sender2);
+
+            Block<DumbAction> b1 =
+                TestUtils.MineNext(receiver.BlockChain.Genesis, difficulty: 1024);
+
+            try
+            {
+                await BootstrapAsync(sender1, receiver.AsPeer);
+                await BootstrapAsync(sender2, receiver.AsPeer);
+
+                sender1.BlockChain.Append(b1);
+                sender2.BlockChain.Append(b1);
+
+                sender1.BroadcastBlock(b1);
+                sender2.BroadcastBlock(b1);
+
+                // Make sure that receiver swarm only filled once for same block
+                // that were broadcasted simultaneously.
+                await receiver.BlockReceived.WaitAsync();
+
+                // Awaits 1 second because receiver swarm may tried to fill again after filled.
+                await Task.Delay(1000);
+                var transport = receiver.Transport as NetMQTransport;
+                Assert.NotNull(transport);
```

Comment:
I guess we don't have to check this assertion here because of `receiver.Transport is NetMQTransport` isn't related to this case.